### PR TITLE
Add possibility to define timeout on client

### PR DESCRIPTION
**Pull request by caspertimmers**
Solves #25

### CODE REVIEW

**Comment by willemstuursma** [comment]
The time out should default to 10 seconds.

File: tests/test_client.py [hunk 1]
```diff
@@ -4,8 +4,10 @@
 import sys
 from datetime import datetime
 
+import mock
 import pkg_resources
 import pytest
+import requests
```
Comment (whyscream) [inline tests/test_client.py]: unused import?

File: tests/test_client.py [hunk 2]
```diff
@@ -240,5 +244,28 @@ def test_client_unicode_error_py3(client, response):
...
+    client.payments.list()
+    assert request_mock.call_args[1]['timeout'] == 300
+
+
+@mock.patch('mollie.api.client.requests.request')
+def test_client_request_timed_out(request_mock, client):
+    """Timeout should raise a RequestError."""
+    request_mock.side_effect = requests.exceptions.ReadTimeout(
+        "HTTPSConnectionPool(host='api.mollie.com', port=443): Read timed out. (read timeout=5)")
+    with pytest.raises(RequestError) as err:
```
Comment (whyscream) [inline tests/test_client.py]: add empty line.

File: tests/test_client.py [hunk 3]
```diff
@@ -240,5 +244,28 @@ def test_client_unicode_error_py3(client, response):
...
+    client.set_timeout(300)
+    client.payments.list()
+    assert request_mock.call_args[1]['timeout'] == 300
+
+
+@mock.patch('mollie.api.client.requests.request')
+def test_client_request_timed_out(request_mock, client):
+    """Timeout should raise a RequestError."""
+    request_mock.side_effect = requests.exceptions.ReadTimeout(
+        "HTTPSConnectionPool(host='api.mollie.com', port=443): Read timed out. (read timeout=5)")
```
Comment (whyscream) [inline tests/test_client.py]: Set the read timeout value in the error response to what we use by default: `(read timeout=10)`

File: tests/test_client.py (hunk 1 above)
Comment (caspertimmers) [reply tests/test_client.py]: requests is on line 267 for `requests.exceptions.ReadTimeout`

**Pull request merged**